app/src/main/java/org/smartregister/sample/fp/app/FPApplication.kt

```diff
@@ -0,0 +1,60 @@
+package org.smartregister.sample.fp.app
+
+import com.evernote.android.job.JobManager
+import org.smartregister.BuildConfig
+import org.smartregister.Context
+import org.smartregister.CoreLibrary
+import org.smartregister.configurableviews.ConfigurableViewsLibrary
+import org.smartregister.repository.Repository
+import org.smartregister.sample.fp.job.FPJobCreator
+import org.smartregister.sample.fp.repository.FPRepository
+import org.smartregister.util.Log
+import org.smartregister.view.activity.DrishtiApplication
+
+open class FPApplication : DrishtiApplication() {
+
+    private var mPassword: String? = null
+
+    override fun onCreate() {
+        super.onCreate()
+
+        mInstance = this
+        context = Context.getInstance()
+        context.updateApplicationContext(applicationContext)
+
+        //Initialize Modules
+        CoreLibrary.init(context, FPConfiguration(), BuildConfig.BUILD_TIMESTAMP)
+        ConfigurableViewsLibrary.init(context)
+
+        //init Job Manager
+        JobManager.create(this).addJobCreator(FPJobCreator())
```

Comment:
Initialise Flurry analytics here if this is going to be actively used OR create an issue to add it later